tools/bridge-deploy/tests/test_cli.py

```diff
@@ -10,18 +10,87 @@ def runner():
     return CliRunner()
 
 
-def test_deploy_foreign(runner):
+def test_deploy_foreign(runner, abitrary_address):
+    result = runner.invoke(
+        main, args=f"deploy-foreign --jsonrpc test --token-address {abitrary_address}"
+    )
+
+    assert result.exit_code == 0
+
+
+def test_print_foreign(runner):
```

Comment:
Strong Suggestion: This is still not implemented, right? Maybe just remove the print function & test alltogether?

Reply:
Sound good. Just don't wanted to remove something not related to this PR.

Reply:
:heavy_check_mark: